client/my-sites/email/add-mailboxes/index.tsx

```diff
@@ -358,14 +366,14 @@ const AddMailboxes = ( props: AddMailboxesProps ): JSX.Element | null => {
 
 	const goToEmail = (): void => {
 		let url = emailManagement(
-			selectedSite.slug,
+			selectedSite?.slug,
```

Comment:
`emailManagement()` already supports an undefined site slug so this just prevents a fatal error if there is no selected site.